src/bootstrap/app-router.js

```diff
@@ -1,190 +0,0 @@
-import React, { useEffect, useMemo, useState } from 'react'
-import { Route, Switch, Redirect } from 'react-router-dom'
-import { useHistory } from 'react-router'
-import { ApolloClient, ApolloProvider, InMemoryCache } from '@apollo/client'
-import { HttpLink } from '@apollo/client/link/http'
-import { useWeb3Context } from 'web3-react'
-import getNetworkEnv from 'utils/network-env'
-import loadable from '@loadable/component'
-import ErrorPage from 'pages/error-page'
-import NoWeb3Detected from 'pages/no-web3'
-import Loading from 'components/loading'
-import connectors from 'config/connectors'
-import { DEFAULT_NETWORK } from 'config/networks'
-import { hexlify, SAVED_NETWORK_KEY } from 'utils/string'
-import { TCR_EXISTENCE_TEST } from 'utils/graphql'
-
-const ItemsRouter = loadable(
-  () => import(/* webpackPrefetch: true */ 'pages/items-router'),
-  { fallback: <Loading /> }
-)
-
-const ItemDetailsRouter = loadable(
-  () => import(/* webpackPrefetch: true */ 'pages/item-details-router'),
-  { fallback: <Loading /> }
-)
-
-const Factory = loadable(
-  () => import(/* webpackPrefetch: true */ 'pages/factory/index'),
-  { fallback: <Loading /> }
-)
-
-const ClassicFactory = loadable(
-  () => import(/* webpackPrefetch: true */ 'pages/factory-classic/index'),
-  { fallback: <Loading /> }
-)
-
-const AppRouter = () => {
-  const history = useHistory()
-  const { networkId, error, account } = useWeb3Context()
-  const isUnsupported = useMemo(() => error?.code === 'UNSUPPORTED_NETWORK', [
-    error
-  ])
-  const [pathResolved, setPathResolved] = useState(false)
-  const [invalidTcrAddr, setInvalidTcrAddr] = useState(false)
-
-  const tcrAddress = getNetworkEnv('REACT_APP_DEFAULT_TCR_ADDRESSES', networkId)
-
-  const client = useMemo(() => {
-    if (!networkId) return null
-
-    const GTCR_SUBGRAPH_URL = getNetworkEnv('REACT_APP_SUBGRAPH_URL', networkId)
-
-    if (!GTCR_SUBGRAPH_URL) return null
-
-    const httpLink = new HttpLink({
-      uri: GTCR_SUBGRAPH_URL
-    })
-    return new ApolloClient({
-      link: httpLink,
-      cache: new InMemoryCache()
-    })
-  }, [networkId])
-
-  // changes network for users without a provider
```

Comment:
> changes network for users without a provider

You didn't copy this functionality to the new `app-router.tsx`
So now if you access `/tcr/100/...` on incognito, without a provider, without having changed chains, you will crash the page. Get this code back.
You can see it for yourself
http://localhost:3000/tcr/100/0x2442D40B0aeCad0298C2724A97F2f1BbDF2C2615